scripts/stage-release.sh

```diff
@@ -210,7 +210,11 @@ gh repo set-default "$(git remote get-url "${REPOSITORY}")"
 echo "Creating draft release notes."
 API_COMPATABILITY_REPORT=kroxylicious-api/target/japicmp/"${RELEASE_VERSION}"-compatability.html
 cp kroxylicious-api/target/japicmp/japicmp.html "${API_COMPATABILITY_REPORT}"
-gh release create --title "${RELEASE_TAG}" --notes-file "CHANGELOG.md" --draft "${RELEASE_TAG}" ./kroxylicious-*/target/kroxylicious-*-bin.* "${API_COMPATABILITY_REPORT}"
+RELEASE_NOTES_DIR=releaseNotes
+mkdir -p ${RELEASE_NOTES_DIR}
```

Comment:
should we gitignore this?

Reply:
I suppose, its currently after all the git operations that would care. It could just as easily be in `/tmp/` I just didn't want to worry about permissions etc in the actions.

Reply:
yeah I just figure it's just a caution thing to save the next person who wants to insert `git add xyz` later, we could also rm it straight after the release command

Reply:
happy either way since it targets a controlled/disposable env as you said

Reply:
I've made it a hidden dir and added it to `.gitignore`